### PR TITLE
Feature/#2901 add a docker arm64 image

### PR DESCRIPTION
**Pull request by YuaFox**
**Dockerfile now compiles the Scada-LTS war file**
This makes it possible to build the image without needing to configure the environment.

I added docker-multiarch.sh script to build the image for any arch. Documented how to use it on the README.md.

Modified workflows to create amd64 and arm64 docker image automatically.

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -178,9 +176,10 @@ jobs:
         with:
           images: scadalts/scadalts
       - name: Build and push Docker image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v5
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
```
Comment (Limraj) [inline .github/workflows/master.yml:188]: Isn't it enough to set this parameter? I understand that the simplest solution is for a single runner and this will lengthen the process of adding individual images, but we want to limit the number of changes to at least necessary to avoid potential problems later. https://docs.docker.com/build/ci/github-actions/multi-platform/

Comment (YuaFox) [reply .github/workflows/master.yml:188]: Yes, that's correct, if you want to limit the changes made you only need to setup QEMU, buildx and then build-and-push with that parameter

**Comment by YuaFox** [comment]
I changed my PR to make the minimum changes possible

File: docker-compose-arm.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Docker compose file to manage your deployed images.
+# Use MySQL server 5.7 and latest Scada-LTS local build.
+# Using attached webapps folder as developer you will be able to modify the static content from host os.
+# Attach shell to stop the tomcat instance and then you will be able to run in JPDA mode.
+version: '3'
+services: 
+    database:
+        container_name: mysql
+        image: biarms/mysql:5.7
```
Comment (Limraj) [inline docker-compose-arm.yml]: 1. Is this file necessary? Docker itself will select the image of the appropriate architecture, based on the host architecture;
2. We use MySQL server version +8;
3. We should use official images from https://hub.docker.com/r/mysql/mysql-server/tags
and here linux/arm64 is an alias for linux/arm64/v8;

Comment (YuaFox) [reply docker-compose-arm.yml]: I get this error when using mysql:8.0 at startup

image
[scadalts-docker-logs.txt](https://github.com/user-attachments/files/15532351/scadalts-docker-logs.txt)

official mysql 5.7 image does not have an arm64 version

Comment (Limraj) [reply docker-compose-arm.yml]: 1. We no longer support mySQL 5.7. Use min version 8.0.32. Is corrected docker-compose file for MySQL 8: https://github.com/SCADA-LTS/Scada-LTS/blob/develop/docker-compose.yml; 
2. What computer are you testing this on?

Comment (YuaFox) [reply docker-compose-arm.yml]: Perfect, it works with that mysql version. I tested this build with a Raspberry Pi 4B 4GB.

I removed my docker-compose example since it works with the "generic" but i changed the comment mentioning mysql 5.7

**Comment by Limraj** [comment]
Hi @YuaFox, please contact us at scada-lts@softq.pl, we need to have some contact with you. (with a link to your github account so you can be located more easily)
For now, let's do it on the branches I created: 
https://github.com/SCADA-LTS/Scada-LTS/pull/2906, 
https://github.com/SCADA-LTS/Scada-LTS/pull/2909. 
Let me know if these changes look ok and if we can merge it.

**Comment by YuaFox** [comment]
> Hi @YuaFox, please contact us at [scada-lts@softq.pl](mailto:scada-lts@softq.pl), we need to have some contact with you. (with a link to your github account so you can be located more easily) For now, let's do it on the branches I created: #2906, #2909. Let me know if these changes look ok and if we can merge it.

Hello, I wrote an email to [scada-lts@softq.pl](mailto:scada-lts@softq.pl). I reviewed the 2 pull requests, they look ok.

**Comment by Limraj** [comment]
@YuaFox Thanks you for your help. 
So I'm closing this pr.
